app/src/main/java/org/sopt/and/feature/home/HomeState.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.and.feature.home
+
+import org.sopt.and.feature.home.model.HomeContent
+import org.sopt.and.feature.home.model.HomeSliderImage
+import org.sopt.and.feature.home.model.getHomeContent
+import org.sopt.and.feature.home.model.getHomeSliderImage
+import org.sopt.and.feature.home.model.getHomeTop20Image
+
+data class HomeState(
+    val pagerImageList: List<HomeSliderImage> = getHomeSliderImage(),
+    val homeContentList: List<HomeContent> = getHomeContent(),
+    val top20List: List<String> = getHomeTop20Image()
```

Comment:
조심스럽게 `get~~()` 부분들을 viewmodel에서 다루는 방법이 좀더 mvvm 구조에 적합하지 않을까? 하는 생각을 남겨봅니다!

Reply:
get ~~()을 사용하면 어떤 이점이 있나요? @wjdrjs00

Reply:
그냥 데이터 넣지말고 함수로 처리하기..메모.. 가독성이 정말 좋네요

Reply:
> 조심스럽게 get~~() 부분들을 viewmodel에서 다루는 방법이 좀더 mvvm 구조에 적합하지 않을까? 하는 생각을 남겨봅니다!

어떤 부분에서 MVVM스럽다는것일까요? @wjdrjs00

Reply:
음,, 제 생각은 추가되는 레이어 없이 현재 프로젝트 구조에서 생각을 했을때, 기존의 HomeState에서 직접 데이터를 가져오는 방식보단 HomeState를 데이터를 보관하는 역할로만 사용하고 실제 데이터를 가져오는 로직을 viewmodel에서 다루는 방법이 좀더 mvvm 관점이 아닐까? 하는 생각이였습니다!
```suggestion
    val pagerImageList: List<HomeSliderImage> = emptyList(),
    val homeContentList: List<HomeContent> = emptyList(),
    val top20List: List<String> = emptyList()
```